### PR TITLE
Test adding Infura to the MetaMask docs

### PR DESCRIPTION
**Pull request by bgravenorst**
Test adding Infura to the MetaMask docs

### CODE REVIEW

File: services/concepts/failover-protection.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Failover protection is available on the Mainnet network for customers on the Growth or Custom plans.
+This feature ensures API requests are fulfilled even if Infura experiences a service issue or
+outage. This is achieved by routing requests to a select partner.
+
+Failover protection is enabled on a per-request basis by adding the failover header to your API
+request (see [Enable API request forwarding](../how-to/enable-api-forwarding.md)). This gives
+you control over which requests you'd like fulfilled by Infura's partner if failover protection should be needed.
+
+## Partners and privacy policies
```
Comment (bgravenorst) [inline services/concepts/failover-protection.md:22]: As a general topic this needs updating. For example, we'll need to list which networks supports Failover.
Also, different partners support different failover networks.

Comment (joaniefromtheblock) [reply services/concepts/failover-protection.md:22]: Thank you @bgravenorst. There are several sections where the direct transfer of information from the Infura docs to MetaMask requires some refinement (or simply doesn't fit well). I started a Google doc when I was doing the restructure, to identify content that may need some "MetaMasking". Maybe we can review it as a team and potentially create some tickets to tackle issues in the future.  For instance, somethings I mention are related to: 

- How we present Infura within MetaMask (RPC nodes for MM)
- Which tutorials are relevant to a user (I'm not sure, but some of the Infura tutorials look like they don't have much to do with MM)
- Where and when to add  testnet documentation (it's presently omitted in MM etc. 

I think we will just iteratively work on it, once the initial doc set is merged.

File: services/concepts/failover-protection.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+description: Learn about failover protection
+sidebar_position: 1
+---
+
+# Failover protection
+
+:::info
+
+Failover support is available on Mainnet only.
```
Comment (bgravenorst) [inline services/concepts/failover-protection.md:10]: This is currently incorrect and misleading so should be updated before merging. We should list the correct networks that support failover. Additionally, perhaps move the "Partners and privacy policies" content to the "services/reference/[NETWORK]/index.md file for the appropriate networks.

Note: Networks that support failover have different partners so it cannot be generic.

File: services/reference/ipfs/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+description: Gas network documentation
```
Comment (bgravenorst) [inline services/reference/ipfs/index.md:2]: Missing the network description

Comment (bgravenorst) [reply services/reference/ipfs/index.md:2]: I meant to say `description` is incorrect, and the topic requires a description for what IPFS is/does etc.

**Review by m4sterbunny — DISMISSED** [review]
as per @bgravenorst approval in Slack

**Review by m4sterbunny — APPROVED** [review]
all tests pass

File: package.json [hunk 4]
```diff
@@ -34,7 +34,8 @@
     "react-dom": "^18.0.0",
     "react-player": "^2.13.0",
     "remark-codesandbox": "^0.10.1",
-    "remark-docusaurus-tabs": "^0.2.0"
```
Comment (Rahul-cyber26) [inline package.json]: #1877

**Pull request merged**